docs/rules/terraform_module_pinned_source_is_semver.md

```diff
@@ -0,0 +1,54 @@
+# terraform_module_pinned_source_is_semver
```

Comment:
Please also add it to the documentation index :)
https://github.com/terraform-linters/tflint/blob/629009c1430cbb92bcd7b79d7b7150ae6b2b01d2/docs/rules/README.md